routes/index.js

```diff
@@ -1,9 +1,12 @@
 const express = require("express");
 const router = express.Router();
+const authMiddleware = require("../middlewares/authenticateUser");
```

Comment:
```suggestion
const authenticateUser = require("../middlewares/authenticateUser");
```
@Behramkhann , it is a good practice to use a middleware with its own name.